mvrx-navigation/src/test/kotlin/com/airbnb/mvrx/navigation/MavericksExtensionsTest.kt

```diff
@@ -76,33 +76,4 @@ class MavericksExtensionsTest {
                 }
         }
     }
-
```

Comment:
@marukami I'm not sure why this stopped throwing the expected exception but things still seem to be working okay. Can you verify?

Reply:
recreate seems to wait for the resume state. Yer, seems to be fine. I'm not sure how to test the exception been thrown.